packages/client/src/containers/Router/routes.ts

```diff
@@ -2,4 +2,6 @@ export enum Routes {
   Login = '/login',
   Forum = '/forum',
   Registration = '/registration',
+  Start = '/start',
+  End = '/end',
```

Comment:
Советую добавить ещё один роут для Home
```suggestion
  End = '/end',
  Home = '/,
```